ios/CollaboraOnlineWebViewKeyboardManager/COKbdMgrTest/test.html

```diff
@@ -168,7 +168,7 @@
           i.src = document.location.href.replace(/\?depth=\d+$/, "") + "?depth=" + (depth + 1);
           i.width = innerWidth - 100;
           i.height = innerHeight - 100;
-          i.allow = "clipboard-read *; clipboard-write *";
+          i.allow = "clipboard-read *; clipboard-write *; fullscreen *";
```

Comment:
is this necessary / supported on iOS?

Reply:
It is not supported indeed according to mdn.

`clipboard-read` and `clipboard-write` are not even on mdn, and seem like chrome-only and not even listed in mdn.

But canIuse has them:
https://caniuse.com/?search=clipboard-write https://caniuse.com/?search=clipboard-read

Reply:
We could remove `clipboard-read` `clipboard-write`, but at the same it does not have much of an impact.

Reply:
Was added in b8a3cae7d2d4a886d617b11d12e43b58e9b6bc8a